i18n/en-US.properties

```diff
@@ -1168,6 +1168,8 @@ boxui.unifiedShare.collaboration.userCollabText = User
 boxui.unifiedShare.collaboratorListTitle = People in '{itemName}'
 # This string is displayed as tooltip on hovering over expire icon for collab
 boxui.unifiedShare.collaborators.expirationTooltipClickableText = Access expires on {date}. Click for details.
+# Error message when more than the maximum number of contacts is entered
+boxui.unifiedShare.contactsExceedLimitError = Oops! You've reached the maximum number of collaborators that can be added at once ({maxContacts} collaborators). Please try again by splitting your invitation into batches.
```

Comment:
Can we change the wording to `Oops!  The maximum number of collaborators that can be added at once is {maxContacts} collaborators.  Please try again by splitting your invitations into batches.`.  I think it's more easy to localize than having something in parentheses.